genesyscloud/provider/provider_sdk_debug.go

```diff
@@ -0,0 +1,74 @@
+package provider
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/google/uuid"
+	"net/http"
+)
+
+type sdkDebugRequest struct {
+	DebugType        string `json:"debug_type,omitempty"`        //Indicates whether it is a request or response debug
+	TransactionId    string `json:"transaction_id,omitempty"`    //Unique id to link the request and response
+	InvocationCount  int    `json:"invocation_count,omitempty"`  //Number of times the URL has been invoked.  Will be greater then zero when it is a retry
+	InvocationMethod string `json:"invocation_method,omitempty"` //HTTP method that will be invoked
+	InvocationUrl    string `json:"invocation_url,omitempty"`    //HTTP URL
+}
+
+func (s *sdkDebugRequest) ToJSON() (err error, jsonStr string) {
+	jsonData, err := json.Marshal(s)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Should we be using `fmt.Println` here? `log` would probably be more appropriate

Reply:
fixed